modules/openapi-generator/src/main/resources/dart2/enum.mustache

```diff
@@ -39,7 +46,7 @@ class {{classname}}TypeTransformer {
       case {{{value}}}: return {{classname}}.{{{name}}};
         {{/enumVars}}
       {{/allowableValues}}
-      default: throw('Unknown enum value to decode: $data');
+      default: return null;
```

Comment:
@agilob would it be better to throw an error rather than returning null?

Reply:
Why would this be better?

Reply:
I actually convinced myself that if enum value is not known, project should be regenerated, but crashing whole deserialisation seems too brutal too. Reverted it.

Reply:
> Why would this be better?

The user won't be notified of the issue (incorrect enum value).

(null could be a possible value in the enum but don't think the Dart client supports it at the moment)